// @thirdweb-dev/react/evm/dist/thirdweb-dev-react-evm.browser.esm.js
export { e as useIsWalletModalOpen, f as useSetIsWalletModalOpen } from '../../dist/headlessConnectUI-3fa33622.browser.esm.js';
export { u as useSafe } from '../../dist/useSafe-c51d29cc.browser.esm.js';
export { u as useMagic } from '../../dist/useMagic-969e28f5.browser.esm.js';
export { C as ConnectWallet, M as MediaRenderer, N as NetworkSelector, h as ThirdwebNftMedia, T as ThirdwebProvider, W as Web3Button, b as useCoinbaseWallet, u as useInstalledWallets, a as useMetamask, d as usePaperWallet, c as usePaperWalletUserEmail, g as useResolvedMediaType, e as useWalletConnect, f as useWalletConnectV1 } from '../../dist/useWalletConnect-a9836c7e.browser.esm.js';
export * from '@thirdweb-dev/react-core';
import '../../dist/basic-72632cb3.browser.esm.js';
import '@emotion/react';
import '@emotion/styled';
import 'react/jsx-runtime';
import '@radix-ui/colors';
import '@thirdweb-dev/wallets';
import 'detect-browser';
import 'react';
import 'react-qr-code';
import '@radix-ui/react-dialog';
import '@radix-ui/react-icons';
import '@tanstack/react-query';
import '@radix-ui/react-popover';
import 'copy-to-clipboard';
import '@radix-ui/react-tooltip';
import 'ethers';
import '@radix-ui/react-tabs';
import 'fuse.js';
import '@radix-ui/react-dropdown-menu';
import '@thirdweb-dev/chains';
import 'tiny-invariant';
